tests/integration-tests/tests/scheduler_plugin/test_scheduler_plugin.py

```diff
@@ -176,7 +179,7 @@ def _start_compute_node(ec2_client, region, cluster, command_executor):
     # Get one launch template for compute node
     lt = _get_launch_templates(command_executor)[0]
     run_instance_command = (
-        f"aws ec2 run-instances --region {region}" f" --count 1" f" --launch-template LaunchTemplateId={lt}"
+        f"aws ec2 run-instances --region {region}" f" --count 1" f" --launch-template LaunchTemplateId={lt},Version=2"
```

Comment:
nit: you can probably do Version=$Latest

Reply:
Changed